sys/Makefile

```diff
@@ -95,6 +95,12 @@ endif
 ifneq (,$(filter ng_ndp,$(USEMODULE)))
     DIRS += net/network_layer/ng_ndp
 endif
+ifneq (,$(filter ng_ndp_hst,$(USEMODULE)))
+    DIRS += net/network_layer/ng_ndp/hst
+endif
+ifneq (,$(filter ng_ndp_rtr,$(USEMODULE)))
+    DIRS += net/network_layer/ng_ndp/rtr
```

Comment:
Functions (and if you wish files) can still be prefixed ..._rtr_... in order to shorten the names.